frontend/src/api/apiMergeUtils.ts

```diff
@@ -0,0 +1,8 @@
+import { QueryParams } from '@openshift/dynamic-plugin-sdk-utils';
+import { K8sAPIOptions } from 'k8sTypes';
```

Comment:
```suggestion
import { K8sAPIOptions } from '../k8sTypes';
```

Reply:
both should work as k8sTypes is at the root of src.

Reply:
Okay, fine -- I'll add it to the linter ticket. This is confusing because without a leading character, it looks like an npm dep.